### ISSUE TITLE
Tac LS - Substrate?

### ISSUE THREAD

**Issue by OakTree42** [BobPalmer/MKS#723]
In the old system there was a way to use substrate and H2O to increase life support quantities. There seems to be no way to do that in the current system. It is very straight forward now. Waste + Fertilizer = food/water/h2O at some net loss even with a level 5 scientist.

Would a system that let you convert water and substrate into Fertilizer ruin the USI-LS systems? Or could something like that be added to perhaps the MKS/OKS ag module? Or perhaps the hab modules to give them a real purpose again? Not sure... just a thought. The Tac LS numbers do need some tweaking. I get the net loss concept, but my testing says that it is a net loss on most areas even with a level 5 scientist, and much more loss with lower levels making it harder to make it useful in career mode, or perhaps I'm reading the numbers wrong?

**Comment by jd284** [comment]
The Cultivator still exists on the aeroponics parts, but it also needs fertilizer now (Water + Subtrate + Fertilizer = Supplies).

**Comment by OakTree42** [comment]
Ah yes, but now everything uses fertilizer, at least everything with anything close to an even return? And we have no way to make fertilizer... is the intent to require fertilizer shipments from Kerbin to a colony on say Eve or Duna to keep them alive? Just curious.... Also, thanks, I had missed that sub part, in fact I was pretty sure I looked there before but after that post JD284, I looked again and... yep, it was there. Must have missed that bit earlier.
